landscape-report.md

```diff
@@ -3,15 +3,15 @@ title: "Sharing Government Software"
 ---
 
 # Sharing Government Software
+*Note: Earlier this year, we changed our name from the State Software Collaborative to the Intergovernmental Software Collaborative. Now, we're changing what we call the groups of governments working in this way from cooperatives to collaboratives. These changes are reflected here.
```

Comment:
thanks! just a few requests:

- Don't forget to add the * at the end of the line :) It won't be italics unless there's an * at the beginning *and* end. 
- Can you link "changed our name" to this update post: https://softwarecollaborative.org/2022/01/24/hypothesis.html
- likewise, can you link "cooperatives to collaboratives" to this post: https://softwarecollaborative.org/2022/10/03/language-update.html